internal/templates/index.go.html

```diff
@@ -1,13 +1,16 @@
-<!-- Copyright (c) 2024 Andrea Goh. -->
+<!-- Copyright (c) 2024 Andrea Goh and Seoyoung Cho. -->
 
 {{ template "title" }}Landing{{ template "end_title" }}
-<div class="container">
-    <h1 style="color:#505c45;">Welcome to MATCHA</h1>
-    <a href="/signup">
-        <button class="btn1" type="button">Signup</button>
-    </a>
-    <a href="/login">
-        <button class="btn2" type="submit">Login</button>
-    </a>
-</div>
+{{ template  "navbar" . }}
+<body>
+<section class="px-8 py-24">
+    <div class="container mx-auto text-center">
+        <h1 class="block antialiased tracking-normal font-sans text-4xl font-semibold leading-[1.3] text-accent-700
+        mb-4">
+            Welcome to MATCHA !</h1>
```

Comment:
I think either make it all in one line or put the </h1> on the next line

Reply:
Even tho its against the formatting?